docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,59 @@
+# How to test App:
+- Using Physical device and apk
+- Using Emulator and Apk
+- Directly Running in Android Studio ( Recommended )
+
+
+## For option one or two:
+
+Every time a new commit is merged into the `develop` branch an _apk_ is released in the `apk` branch. To Navigate to `apk` Branch see here: 
+
+![Branch chnage](https://user-images.githubusercontent.com/16835617/92631953-8a52a900-f2ef-11ea-9e61-60692555283b.png)
+
+
+If you want to do manual testing on the application without setting up the development environment, you can download this [app-debug.apk](https://github.com/anitab-org/mentorship-android/blob/apk/app-debug.apk) and install it on your Android device.
```

Comment:
can you please make a change in line 14 from "manual testing on the application" to "manual testing of the application ?